mmaction/models/heads/base.py

```diff
@@ -3,7 +3,7 @@
 import torch
 import torch.nn as nn
 
-from ...core import top_k_accuracy
+from ...core.evaluation.accuracy import top_k_accuracy
```

Comment:
why?

Reply:
That confuses me too. Check [this CI log](https://github.com/open-mmlab/mmaction2/pull/133/checks?check_run_id=993420247#step:10:40) out. The only difference is [this](https://github.com/open-mmlab/mmaction2/pull/133/commits/18343a55a99a79dda45e1af9674e0cd876ea5c55).

Reply:
I guess it is what's called 'circular import'. When import all from `.evaluate`, it imports `eval_hook`. But in eval_hook it imports `mmaction.models`, which imports `head.py` in which it imports `evaluate`. Forms a circle!

Reply:
It was caused by a circular importing. You can try to move it inside `def after_train_epoch(self, runner):` (like `from mmaction.apis import single_gpu_test`)

Reply:
So I either write `from ...core.evaluation.accuracy import top_k_accuracy`, or in eval_hooks.py write `from mmaction.models.common import SubBatchBN3d` to avoid importing head.py. Both ugly.

Reply:
Moving it inside `aggregate_sub_bn_stats` will cause repeatly importing it, hurting the efficiency. But it will only run for like 53 times in i3d, is it acceptable?

Reply:
Yeah, I think it is acceptable.